### ISSUE TITLE
Icon Scan Overlay does not stretch icons to scale

### ISSUE THREAD

**Issue by AkiraChisaka** [RatScanner/RatScanner#389]
**Describe the bug**
It seems like the on screen display generated by the RatScanner Overlay does not work properly in a multi-monitor setup. It may also be caused by windows resolution scaling?
A simple fix could be just have a setting in the app to disable the overlay.

**To Reproduce**
For me, simply using RatScanner while the overlay is enabled cause it to not function as intended. See screenshots below for more info.

**Expected behavior**
Expected the overlay will work as the Github readme GIFs shown.

**Screenshots**
![image](https://user-images.githubusercontent.com/47231500/177054631-dd31b475-aaf6-4784-a7b0-a1e296a0d2a9.png)
For example, in this image above, you can see the scanned result is on the bottom right, while the item is on the top left.

![image](https://user-images.githubusercontent.com/47231500/177054649-fe379d99-a709-4a72-96d6-379881b6b3e4.png)
The same thing happens with clicking the magnify glass to inspect. The currency is really far away from where I clicked.

**Additional context**
Here are my graphics settings, which may be relevant.
![image](https://user-images.githubusercontent.com/47231500/177054788-0d485f24-c57a-44ee-a196-6410e5d87356.png)
![image](https://user-images.githubusercontent.com/47231500/177054811-a5827760-9638-4c34-97a6-78caedf9782c.png)

**Log file content**
```
[19:24:45.0459184] > [Info]  ----- RatScanner 3.2.3 -----
[19:24:45.0463019] > [Info]  Starting RatScanner...
[19:24:45.0463049] > [Info]  Checking for updates...
[19:24:45.0496091] > [Info]  Loading resource "RSClientVersion"...
[19:24:45.4681945] > [Info]  A new version is available: 3.3.1
[19:24:45.4902260] > [Info]  Loading config...
[19:24:45.4912943] > [Info]  Checking for item data updates...
[19:24:45.4921318] > [Info]  Loading resource "RSItemDataBundleVersion"...
[19:24:45.5625095] > [Info]  Loading TarkovTracker...
[19:24:46.6425261] > [Info]  Loading progress data...
[19:24:46.8524891] > [Info]  Loading price data...
[19:24:47.1617697] > [Info]  Initializing RatEye...
[19:24:49.9952526] > [Info]  Setting default item...
[19:24:50.0579459] > [Info]  Initializing hotkey manager...
[19:24:50.0597843] > [Info]  Setting up data update routines...
[19:24:50.0598003] > [Info]  Ready!
[19:29:50.0717987] > [Info]  Refreshing TarkovTracker DB...
```

**Comment by Blightbuster** [comment]
Hey, did you set the scaling in rat scanners general settings to the same as in windows?

**Comment by AkiraChisaka** [comment]
> Hey, did you set the scaling in rat scanners general settings to the same as in windows?

Hmm, I don't think I saw a scaling settings?
![image](https://user-images.githubusercontent.com/47231500/177062893-6d73e22d-a61c-4755-9b07-752e72b4f3bb.png)

Did I download an out of date version? I downloaded from the Github release page I think.

**Comment by AkiraChisaka** [comment]
Oh cool there are updates, I never updated lol

**Comment by AkiraChisaka** [comment]
![image](https://user-images.githubusercontent.com/47231500/177063910-bfe6ada5-d227-4eb5-a5c5-9b869ed020c1.png)
Ok, after updating it works basically as intended now. Nice!

However, it seems like it's still a tiny bit off... not sure if it can be fixed.
![image](https://user-images.githubusercontent.com/47231500/177063956-42df8261-6eef-4fb8-9eea-fb938c950803.png)
Here are the settings I use.

**Comment by AkiraChisaka** [comment]
Thanks for the edit!

I was out of town traveling lol

Time to get back to Tarkov!

**Comment by Blightbuster** [comment]
Hey, sorry for the late response.
The issue has actually since been fixed in RatScanner v3.3.9 (which was released in September 2022) and all versions onwards.

Thanks for submitting the issue ❤️
